angularjs-portal-home/src/main/webapp/js/app-config.js

```diff
@@ -25,7 +25,7 @@ define(['angular'], function(angular) {
             },
             'notificationsURL' : '/web/samples/notifications.json',
             'groupURL' : '/portal/api/groups',
-            'kvURL' : '/portal/api/keyvalue/'
+            'kvURL' : '/storage/'
```

Comment:
This was more for mock data than anything else, hmmm, how to handle this... We do overlay it in environments.